arrows/ocv/CMakeLists.txt

```
@@ -88,6 +89,7 @@ set( ocv_sources
   feature_detect_extract_ORB.cxx
   feature_detect_extract_SIFT.cxx
   feature_detect_extract_SURF.cxx
+  inpaint.cxx
   image_container.cxx
```

```suggestion
  image_container.cxx
  inpaint.cxx
```

(Given the consistency, I'm curious if this name changed at some point while you were working on this? 😉 )

---

I think I was just consistently bad at the alphabet.